pkg/tfvars/openstack/bootstrap_ignition.go

```diff
@@ -0,0 +1,160 @@
+package openstack
+
+import (
+	b64 "encoding/base64"
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"strconv"
+	"strings"
+
+	ignition "github.com/coreos/ignition/config/v2_2/types"
+	"github.com/gophercloud/gophercloud/openstack/objectstorage/v1/containers"
+	"github.com/gophercloud/gophercloud/openstack/objectstorage/v1/objects"
+	"github.com/gophercloud/utils/openstack/clientconfig"
+	"github.com/vincent-petithory/dataurl"
+)
+
+// CreateBootstrapSwiftObject creates a container and object in swift with the bootstrap ignition config.
+func CreateBootstrapSwiftObject(cloud string, bootstrapIgn string, clusterID string) (string, error) {
+	opts := clientconfig.ClientOpts{
+		Cloud: cloud,
+	}
+
+	conn, err := clientconfig.NewServiceClient("object-store", &opts)
+	if err != nil {
+		return "", err
+	}
+
+	containerCreateOpts := containers.CreateOpts{
+		ContainerRead: ".r:*",
+
+		// "kubernetes.io/cluster/${var.cluster_id}" = "owned"
+		Metadata: map[string]string{
+			"Name":               fmt.Sprintf("%s-ignition", clusterID),
+			"openshiftClusterID": clusterID,
+		},
+	}
+
+	_, err = containers.Create(conn, clusterID, containerCreateOpts).Extract()
+	if err != nil {
+		return "", err
+	}
+
+	objectCreateOpts := objects.CreateOpts{
+		ContentType: "text/plain",
+		Content:     strings.NewReader(bootstrapIgn),
+		DeleteAfter: 3600,
+	}
+
+	objID := strconv.Itoa(rand.Intn(1000000000000000))
```

Comment:
Previously we used a string of 16 characters to generate the url, now it's just a number. I recommend to use https://github.com/kubernetes/apimachinery/blob/master/pkg/util/rand/rand.go#L98 to generate a random string.